msal_extensions/cache_lock.py

```diff
@@ -25,7 +32,29 @@ def __init__(self, lockfile_path):
             flags=portalocker.LOCK_EX | portalocker.LOCK_NB,
             **open_kwargs)
 
+    def try_to_create_lock_file(self):
+        timeout = 5
+        check_interval = 0.25
```

Comment:
Also, let's move that module-level `current_time()` definition to here.
```suggestion
        check_interval = 0.25
        current_time = getattr(time, "monotonic", time.time)
```